lib/Translations.php

```diff
@@ -92,21 +92,18 @@ public function make_pots() {
 	}
 
 	/**
-	 * Wrap jsons with a layer with the key 'locale_data'.
+	 * Apply transformer given the format for the data.
 	 *
-	 * @since  0.0.0
-	 * @param  array $jsons
-	 * @return array
+	 * @since 0.0.0
+	 * @return mixed
 	 */
-	private function wrap_jsons( array $jsons ) : array {
-		foreach ( $jsons as $key => $json ) {
-			$jsons[ $key ] = json_encode(
-				[
-					'locale_data' => json_decode( $json, true ),
-				],
-				JSON_PRETTY_PRINT
-			);
+	private function apply_transformer( $data, Project $config ) {
+		//TODO: receive outside classes.
```

Comment:
This doesn't actually make sense in this way. What we'll have is a string with the class name for the transformer in the config file. So it'll not come as an argument, but will work mostly as is here.

It's more like this (just a rough example):
```php
$transformer_class = $config->get_tranformer() ? $config->get_tranformer() : __NAMESPACE__ . '\\Transformers\\' . ucfirst( $config->get_format() );
```